Library/Homebrew/extend/os/mac/diagnostic.rb

```diff
@@ -264,8 +264,12 @@ def check_for_latest_xquartz
 
         <<-EOS.undent
           Your XQuartz (#{installed_version}) is outdated.
-          Please install XQuartz #{latest_version} (or delete it):
-            https://xquartz.macosforge.org
+          Please install XQuartz #{latest_version} (or delete the current version).
+          XQuartz can be installed via Homebrew-Cask with
+            brew cask install xquartz
```

Comment:
This will always fail. @reitermarkus @jawshooah thoughts on recommending `--force` here to perform an upgrade?